spec/datadog/core/error_spec.rb

```diff
@@ -90,11 +90,11 @@ def call
 
           # Outer-most error first, inner-most last
           wrapper_error_message = /in.*wrapper': wrapper layer \(RuntimeError\)/
-          wrapper_caller = /from.*in `call'/
+          wrapper_caller = /from.*in ['`]call'/
```

Comment:
Ruby 3.4.0-preview2 changes the format of error messages ([see here](https://www.ruby-lang.org/en/news/2024/10/07/ruby-3-4-0-preview2-released/)), so I adjusted the syntax of these rspec tests to consider the new formatting.

Reply:
Lets write a code comment about this.